LanguageFeatures/Augmentation-libraries/augmented_expression_A10_t26.dart

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion Consider a non-augmenting member declaration Dm that occurs
+/// inside an augmenting type declaration Dt. A compile-time error occurs if the
+/// identifier `augmented` occurs in Dm.
+///
+/// @description Checks that it is a compile-time error if an augmenting class
+/// declaration extends a type named `augmented`.
```

Comment:
Perhaps:
```suggestion
/// declaration extends a type that contains the name `augmented`.
```

Similarly for a few more tests below.